frontend/src/shared/components/baseSection/baseSection.tsx

```diff
@@ -28,7 +29,7 @@ export default abstract class BaseSection<T> extends React.Component<BaseSection
                     return (
                         <div className={sectionStyle}>
                             {this.props.data.map((object: T, idx: number) => {
-                                    return this.renderCard(object, idx % 3, language, idx)
+                                    return this.renderCard(object, idx % CardStyleLength, language, idx)
```

Comment:
Minor nitpick, watch for when CardStyleLength is zero. Probably won't affect anything here since we will always have at least one card

Reply:
Yeah, but good catch. Maybe good to have a default card that's completely transparent in base section, and have the message define that value.